testnet-contracts/ADMIN-API.md

```diff
@@ -0,0 +1,11 @@
+# This document describes how to hook eRowan into peggy as a Cosmos Native Asset
+
+## Please follow these instructions to the T. If you do not, peggy may not behave in the way you expect it to with eRowan
+
+1. You will need to have created eRowan as an ERC20 token on the mainnet. The token's symbol should be "PEGGYeRowan"
```

Comment:
can we add cli/scripts for all these? especially for number 4, hardcode it so that there is no risk of mess/new token.

Reply:
Yes. In the eRowan token repo I have hard coded this: 

https://github.com/Sifchain/eRowan-ERC20/blob/master/testnet-contracts/contracts/BridgeToken.sol#L16

Also I'm going to be closing this PR and moving over to this PR which is on the correct branch. https://github.com/Sifchain/sifnode/pull/200